README.md

```diff
@@ -52,6 +52,7 @@ Barista already includes `espresso-core` and `espresso-contrib`. If you need [an
 clickOn(R.id.button);
 clickOn(R.string.button_text);
 clickOn("Next");
+clickOn(R.id.button, "Next");
```

Comment:
Why would you need this method when you already have one for ids and another for strings?

Reply:
It's quite strange to have different Views sharing the same ID but not the same text, at least in our projects 😅
@DeanPike do you have any good example when this `clickOn` is needed or useful? Sometimes, we are tempted to offer this kind of smart and complementary methods, but after writing them, we realize that they are not as useful as we thought and we remove them from the public API.
We try to offer a small API because it's just less code to understand as a user, and less code to maintain as a committer.